adapter/internal/api/apis_impl.go

```diff
@@ -147,7 +238,20 @@ func getAPIType(apiJsn []byte) (string, error) {
 		return "", unmarshalErr
 	}
 	data := apiDef["data"].(map[string]interface{})
-	apiType := data["type"].(string)
+	apiType := strings.ToUpper(data["type"].(string))
```

Comment:
Shall we introduce a constant for "type"

Reply:
Introducing constant for type